src/robusta/core/reporting/url_helpers.py

```diff
@@ -0,0 +1,26 @@
+import logging
+from urllib.parse import ParseResult, parse_qs, urlencode, urlparse
+
+from robusta.core.model.env_vars import ROBUSTA_UI_DOMAIN
+
+
+PROM_GRAPH_PATH_URL: str = "/graph"
+PROM_GRAPH_URL_EXPR_PARAM: str = "g0.expr"
+
+
+def convert_prom_graph_url_to_robusta_metrics_explorer(prom_url: str, cluster_name: str, account_id: str) -> str:
+    parsed_url: ParseResult = urlparse(prom_url)
```

Comment:
Should we wrap the logic within this function, to be more "forgiving" in case of a parsing error?
Atm, it seems that if we have a parsing error, the alert notification won't be sent

Reply:
I don't think it should happens but let's do it anyway, nothing to lose.